### ISSUE TITLE
ERA_Discovery: Faceting

### ISSUE THREAD

**Issue by pbinkley** [ualbertalib/HydraNorth#37]
- faceting: Leah will review ERA's fields and provide specs

**Comment by awakaruk** [comment]
See ERA Facets Google Doc  (questions encouraged) https://docs.google.com/a/ualberta.ca/document/d/18OIW0TQ5T0OGNjuZAgwFRLqBuvW0Xxq-3PynLqhHbfo/edit?usp=sharing

**Comment by criedlberger** [comment]
Need to revisit this code when PCDM becomes part of the codebase.

**Comment by pgwillia** [comment]
@criedlberger did you see this? https://github.com/projecthydra/hydra-collections#make-the-collection-show-as-a-facet-in-your-catalogcontroller

**Comment by pbinkley** [comment]
I've merged #224 , but we should evaluate master now with a test migration and see how it looks with more than a handful of items.

**Comment by weiweishi** [comment]
I haven't created the appropriate rake task tests, and will take a look
tomorrow. I have created other tests for changes I made in the main app's
code. Are you okay with merging with this branch for now, if I can't finish
the tests before Wednesday and pending additional tests for tasks for my
return? Didn't want to hold back on neither the user migration nor the
tests but may not be able to complete both before my vacation.

Weiwei

On Apr 27, 2015, at 5:40 PM, Peter Binkley <notifications@github.com> wrote:

I've merged #224 <https://github.com/ualbertalib/HydraNorth/pull/224> , but
we should evaluate master now with a test migration and see how it looks
with more than a handful of items.

—
Reply to this email directly or view it on GitHub
<https://github.com/ualbertalib/HydraNorth/issues/37#issuecomment-96851415>.
